merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/TypescriptCodeGenerationService.java

```diff
@@ -16,10 +16,15 @@ public String generateTypescriptTypesFromMissionModel(String missionModelId)
   {
     final var activityTypes = this.missionModelService.getActivityTypes(missionModelId);
 
-
-
     final var result = new ArrayList<String>();
     result.add("/** Start Codegen */");
+
```

Comment:
Before this change, was this method just a placeholder?

Reply:
I think so :)

Before this change the method was:
```java
  public static String generateTypescriptTypesFromMissionModel() {
    final var result = new ArrayList<String>();
    result.add("/** Start Codegen */");
    result.add("/** End Codegen */");
    return joinLines(result);
  }

Reply:
Yup, I copied a lot of boilerplate from the scheduling eDSL compiler, including this, in a previous PR. This PR fills in the code gen.